src/M6Web/Component/RedisMock/RedisMock.php

```diff
@@ -333,6 +333,21 @@ public function sadd($key, $members)
 
     }
 
+    public function sdiff($key)
+    {
+        $this->stopPipeline();
+        $keys = is_array($key) ? $key : func_get_args();
+        $result = [];
+        foreach ($keys as $key) {
+            $result[] = $this->smembers($key);
+        }
+        $result = array_values(call_user_func_array('array_diff', $result));
```

Comment:
Since Php7.1 is required, can you use [`...` operator](https://www.php.net/manual/en/functions.arguments.php#functions.variable-arg-list)?
```php
$result = array_values(array_diff(...$result));
```

Reply:
Yes, I can, I tried to stay consistent with `sinter`. But I'll fix that in a separate PR too.